### DISCUSSION TITLE
Certain UBL commands don't function on the UBL firmware installed on a Ender 3 V2

### DISCUSSION THREAD

**Discussion by archaengel** [mriscoc/Ender3V2S1#1230] [Q&A]
Howdy folks!

(I'm using the precompiled UBL firmware)

I'm running through some calibration with my Ender 3 V2, and attempting to issue commands like `G29 P2 B T` and `G29 P4` commands from the [marlin UBL docs](https://marlinfw.org/docs/features/unified_bed_leveling.html) gives me back `?P2 is only available when an LCD is present.` Has anyone else run into this or know what's going on? I'm running the stock display on the printer and I haven't noticed any other issues with it when I'm going through the calibration steps mentioned in the repos wiki.

**Comment by mriscoc** [comment]
For Marlin, LCD means the former Alphanumeric screen with MarlinUI.

**Reply by archaengel** [reply]
Ah dang, that makes sense. Thank you!
